app/css/components/footer-menu.css

```diff
@@ -0,0 +1,52 @@
+/* Footer menu */
+.footer-menu {
+  display: flex;
+
+  width: 100%;
+  height: 100%;
+}
+
+.footer-menu__item {
```

Comment:
Once you're fine with new naming and styles, I'll add note to the readme about the final style.